presto-hive/src/test/java/com/facebook/presto/hive/metastore/TestCachingHiveMetastore.java

```diff
@@ -426,7 +444,7 @@ public void testListRoles()
 
     public void testInvalidGetPartitionsByNames()
     {
-        Map<String, Optional<Partition>> partitionsByNames = metastore.getPartitionsByNames(TEST_METASTORE_CONTEXT, BAD_DATABASE, TEST_TABLE, ImmutableList.of(TEST_PARTITION1));
+        Map<String, Optional<Partition>> partitionsByNames = metastore.getPartitionsByNames(TEST_METASTORE_CONTEXT, getTableOrElseThrow(TEST_DATABASE, TEST_TABLE), ImmutableList.of(BAD_PARTITION));
```

Comment:
This change was made intentionally since this test case ideally should be checking for bad partition rather than checking with bad database. This is the same case in Trino as well.